src/client/cl_predict.c

```diff
@@ -64,9 +63,14 @@ int32_t Cl_PointContents(const vec3_t point) {
 			continue;
 		}
 
-		const int32_t head_node = Cl_HullForEntity(s);
 		const cl_entity_t *ent = &cl.entities[s->number];
 
+		if (ent == cl.entity) {
+			continue;
+		}
+
+		const int32_t head_node = Cl_HullForEntity(s);
+
```

Comment:
This was slightly alarming: `Cl_PointContents` did not skip self! I can't imagine that's a good thing, so it does now.